src/Authentik/Provider.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace SocialiteProviders\Authentik;
+
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    const IDENTIFIER = 'AUTHENTIK';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = ['openid goauthentik.io/api profile email'];
```

Comment:
This scope format looks like an error

Reply:
It's a valid scope format. (tested and working)